packages/calcite-components/src/components/combobox-item/combobox-item.scss

```diff
@@ -3,7 +3,6 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-combobox-item-border-color: Specifies the component's border color.
```

Comment:
Why was this removed? This would be a breaking change.

https://developers.arcgis.com/calcite-design-system/components/combobox-item/#api-reference-styles---calcite-combobox-item-border-color